packages/plans-grid/src/plans-accordion-item/index.tsx

```diff
@@ -112,17 +118,42 @@ const PlanItem: React.FunctionComponent< Props > = ( {
 								) }
 							</div>
 							<div className="plans-accordion-item__price-note">
-								{ isFree
-									? __( 'free forever', __i18n_text_domain__ )
-									: __( 'billed annually', __i18n_text_domain__ ) }
+								{ isFree && __( 'free forever', __i18n_text_domain__ ) }
+
+								{ ! isFree &&
+									( billingInterval === 'ANNUALLY'
+										? createInterpolateElement(
+												__( 'per month, billed as <price /> annually', __i18n_text_domain__ ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'per month, billed annually' )` *ES Score: 8*